.github/workflows/run-regression.yml

```diff
@@ -0,0 +1,35 @@
+name: Call Notification API QA Regression Tests
+
+on: 
+  workflow_call:
+    inputs:
+      environment:
+        description: 'Notification-api deployment environment to test (dev|perf|staging)'
+        required: true
+        type: string
```

Comment:
```suggestion
```
Environment is a specific thing. [Not sure](https://docs.github.com/en/actions/using-jobs/using-environments-for-jobs) it should have type.

Reply:
`type: string` is used in the QA regression.yml file and I don't think it should be removed since it has nothing to do with environments as the API used them. It's literally just a string that gets passed to the --environment flag on the QA Suite.

Reply:
type being changed to `environment`, [per the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype).